.editorconfig

```diff
@@ -0,0 +1,16 @@
+root = true
```

Comment:
I have found [EditorConfig](http://editorconfig.org) to be very helpful as a maintainer because it takes care of nuisances with whitespace and line endings and newlines etc. behind the scenes. e.g. the IntelliJ platform comes with support built in and almost all popular editors have a simple installer. It's worth noting that [GitHub also has native support](https://github.com/editorconfig/editorconfig.github.com/pull/48).

If you still prefer it not be added, I will remove it.